.gitignore

```diff
@@ -23,3 +23,7 @@
 
 # Ignore ReadTheDocs site style
 /docs/site
+
+# Ignore vendor / bundle
+/.bundle/
+/vendor/bundle
```

Comment:
Not sure this is necessary in order for this change to be effective? We're trying to make sure pull requests only commit changes having to do with the intended fix.